torch/_functorch/apis.py

```diff
@@ -101,7 +355,10 @@ def grad(func: Callable, argnums: argnums_t = 0, has_aux: bool = False) -> Calla
         should not depend on the result of a context manager outside of ``f``.
 
     """
+    # To avoid cyclical dependency.
+    import torch._functorch.eager_transforms as eager_transforms
```

Comment:
HACK: If it's ok, will fix this in separate PR.

Reply:
SGTM